tests/compare-sharding.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+
+trap 'kill 0' SIGTERM
+
+kubectl -n kube-system port-forward deployment/kube-state-metrics 8080:8080 &
```

Comment:
Let's move `kube-system` and `kube-state-metrics` to a variable. This way, we are able to correlate the `pod_name` and `pod_namespace` parameters specified in the README with those mentioned in this test.

Reply:
hmm .. how strongly do you feel about this, I quite like the explicitness, and feel introducing variables would make this harder to understand

Reply:
I can see that. Works for me.